edr_server/core/models/extents.py

```diff
@@ -138,7 +138,9 @@ class SpatialExtent(EdrModel["SpatialExtent"]):
 
     def __post_init__(self):
         if not (isinstance(self.bbox, Polygon)):
-            raise(TypeError(f'Expect polygon, received {type(self.bbox)}'))
+            raise(TypeError(f'Expected polygon, received {type(self.bbox)}'))
+        if not (isinstance(self.crs, CrsObject)):
+            raise(TypeError(f'Expected CrsObject, received {type(self.crs)}'))
```

Comment:
I missed this on the Polygon PR, but `raise` is a keyword, rather than a function, so the brackets aren't required and are normally left out